packages/ui/src/layout/banners/LW11CountdownBanner/Countdown.tsx

```diff
@@ -0,0 +1,34 @@
+import Countdown from 'react-countdown'
```

Comment:
not a blocker but UI patterns should probably be not be so specific "LW11CountdownBanner" doesn't sound very reusable

Reply:
yeah can fix in a follow up pr